nibabel/nifti1.py

```diff
@@ -232,6 +233,17 @@
     (2003, 'rgb vector', (), "NIFTI_INTENT_RGB_VECTOR"),
     (2004, 'rgba vector', (), "NIFTI_INTENT_RGBA_VECTOR"),
     (2005, 'shape', (), "NIFTI_INTENT_SHAPE"),
+    # FSL-specific intent codes - FNIRT
```

Comment:
Thanks - or put the 5.09 data in there - approximate should be good enough to find it again.